src/main/java/app/gpx_animator/core/renderer/Renderer.java

```diff
@@ -668,22 +668,17 @@ private Point2D drawMarker(final BufferedImage bi, final int frame) throws UserE
         final var t2 = getTime(frame);
         final var trackConfigurationList = cfg.getTrackConfigurationList();
 
-        var i = 0;
         var trackIdx = 0;
         outer:
-        for (final var timePointMapList : timePointMapListList) {
-            final var trackConfiguration = trackConfigurationList.get(i++);
+        for (int i = 0; i < timePointMapListList.size(); i++) {
+            var timePointMapList = timePointMapListList.get(i);
+            final var trackConfiguration = trackConfigurationList.get(i);
             for (final var timePointMap : timePointMapList) {
                 final var interpolator = interpolators.get(trackIdx++);
                 final var ceilingEntry = timePointMap.ceilingEntry(t2);
                 point = interpolator.getPointAtTime(t2);
                 if (point == null) {
-                    final var floorEntry = timePointMap.floorEntry(t2);
-                    if (floorEntry == null) {
-                        continue;
-                    } else {
-                        point = floorEntry.getValue();
-                    }
+                    continue;
```

Comment:
There was a new feature implemented three weeks ago by @fgaignat (interpolated positions). Won't removing this code damage the new feature?